CameraService.lua

```diff
@@ -384,7 +384,7 @@ function CameraService:SetCameraView(__type: string) --> Used to change views (i
 
 		--> Resets it back to non-scriptable, Roblox default camera
 		self.LockMouse = false
-		cam.CameraSubject = 
+		cam.CameraSubject = nil
```

Comment:
Thanks for catching the typo! I'll re-integrate the previous line I had prior (currentCharacter's Humanoid) to ensure it doesn't falter.